keps/sig-storage/20181111-extend-datasource-field.md

```diff
@@ -130,6 +132,9 @@ Given that the only implementation changes in Kuberenetes is to enable the featu
 
 ## Implementation History
 
+Update to beta status for 1.16 release
```

Comment:
Since we haven't landed beta yet, let's just update this to say that 1.15 is alpha

Reply:
My bad, I thought we needed to update the KEP to indicate our intent to go to beta for 1.16.  I'll replace that, and then when 1.16 lands I'll update the Implementation history then.